comment/templates/comment/comments/content.html

```diff
@@ -4,10 +4,10 @@
     {% include "comment/comments/urlhash.html" with urlhash=urlhash %}
     {% endblock comment_urlhash %}
     {% if text_2 %}
-        {{ text_1|safe }}
```

Comment:
I believe this will break this #169

Reply:
yes, you are right. I have made changes to the `render_content` template tag to first escape the original content and then marked it as safe after our change.